core/jvm/src/main/scala/fs2/compress.scala

```diff
@@ -208,7 +208,9 @@ object compress {
         def push(chunk: Chunk[Byte]): Unit = {
           val arr: Array[Byte] = {
             val buf = new Array[Byte](chunk.size)
-            chunk.copyToArray(buf) // Note: we can be slightly better than this for Chunk.Bytes if we track incoming offsets in abis
+            chunk.copyToArray(
+              buf
+            ) // Note: we can be slightly better than this for Chunk.Bytes if we track incoming offsets in abis
```

Comment:
Weird that a comment is part of the code.. Seems like it made it go over the configured column limit.